cmd/registry-api-descriptor-template/main.go

```diff
@@ -44,7 +45,15 @@ func main() {
 
 	tmpl := template.Must(template.New(filename).Funcs(funcMap).ParseFiles(path))
 
-	if err := tmpl.Execute(os.Stdout, v2.APIDescriptor); err != nil {
+	data := struct {
+		RouteDescriptors []v2.RouteDescriptor
+		ErrorDescriptors []errcode.ErrorDescriptor
+	}{
+		RouteDescriptors: v2.APIDescriptor.RouteDescriptors,
+		ErrorDescriptors: errcode.GetErrorDescriptors(),
```

Comment:
This particular tool is only for v2 registry api. We can make another tool for rendering errors into a template. Imagine something like this:

```
error-template-tool registry.api.v2 daemon < template.tmpl
```